src/styles/blocks/container.scss

```diff
@@ -0,0 +1,22 @@
+.content {
```

Comment:
your block is content but name of file is container.scss, it's better to have same name for file and css block